bundles/org.openhab.binding.linky/pom.xml

```diff
@@ -14,6 +14,10 @@
 
   <name>openHAB Add-ons :: Bundles :: Linky Binding</name>
 
+  <properties>
+    <bnd.importpackage>javax.annotation.meta;resolution:=optional</bnd.importpackage>
+  </properties>
```

Comment:
Why did you add that?

Reply:
It's the way I found to solve the dependency resolution issue I faced.

Reply:
It is building fine without that.

Reply:
@wborn @lsiepel @jlaur  : can you help me to understand why this new property needs to be added ?

Reply:
I will check later if this is something really necessary and if something similar is already done in other bindings but now our priority is to merge to have this fix in the coming milestone.